front/lib/api/assistant/global_agents.ts

```diff
@@ -858,6 +859,69 @@ async function _getDustGlobalAgent(
     };
   }
 
+  const actions: AgentActionConfigurationType[] = [
+    {
+      id: -1,
+      sId: GLOBAL_AGENTS_SID.DUST + "-datasource-action",
+      type: "retrieval_configuration",
+      query: "auto",
+      relativeTimeFrame: "auto",
+      topK: "auto",
+      dataSources: dataSources.map((ds) => ({
+        dataSourceId: ds.name,
+        workspaceId: prodCredentials.workspaceId,
+        filter: { parents: null },
```

Comment:
I removed tags from this cause they were removed on https://github.com/dust-tt/dust/commit/99a08fcea4170c1041f7ce195173e310b579819c